content/downloads/_index.md

```diff
@@ -11,10 +11,15 @@ The table [below](#spire-releases) lists the available releases for [SPIRE](/spi
   * The `spire-agent` and `spire-server` binaries
```

Comment:
Good idea. I've pushed an update that clarifies that the binaries are for Linux.

I've asked Evan Gilman in the past about supported Linux kernels and he has said it would take some research to figure out. Perhaps it's at least an interim fix to specify "Linux".